pandas_sphinx_theme/static/js/theme.js

```diff
@@ -0,0 +1,123 @@
+/******/ (function(modules) { // webpackBootstrap
+/******/ 	// The module cache
+/******/ 	var installedModules = {};
```

Comment:
I suppose this is something autogenerated that we don't need to look at?

Reply:
Correct. This is autogenerated. Or in more frontend terminology, transpiled from modern javascript (which is wat we want to write) into javascript every browser is able to understand, even the older ones.

Reply:
If it is transpiled every time that the theme builds, then should we avoid committing it to the git repository? Or is it still best-practice to commit it?

Reply:
(same w/ the `yarn.lock` file...I'm happy to follow the guidance of others on this, since I'm not a JS dev, but just wanna flag it for discussion)

Reply:
For this theme.js, I think the big advantage of having it commited into the repo, is that you can still do `pip install` from the repo (as everybody is now doing, since there isn't an official release yet) and have a working theme without needing to transpile it yourself (and thus install the js toolchain).

This seems an important use case to keep working.

For yarn.lock, less sure.

We might want to add some check (on CI, or a pre-commit hook, ..) to ensure the transpiling was run? (or in the future)

Reply:
It should indeed be committed.